common/src/main/java/whocraft/tardis_refined/common/GravityUtil.java

```diff
@@ -18,7 +18,7 @@ public class GravityUtil {
     private static final int MAX_Y = 30; // The most a shaft can carry a player up
     private static final double ACCELERATION = 0.2;
     public static boolean isInAntiGrav(Player playerBox, AABB box, Level level) {
-        if(level.dimensionTypeId() != TRDimensionTypes.TARDIS) return false;
+//        if(level.dimensionTypeId() != TRDimensionTypes.TARDIS) return false; //Allow sound to play no matter what dimension the block is placed in
```

Comment:
Is this commented code to remain or to be removed?

Reply:
Could also be a tag?

Reply:
Commented because I was going to ask if it was ok to do so. 

Previously the playing of the sound was restricted to the Tardis dimension.
However, players can craft this block so will be using it outside the Tardis dimension.
For a consistent experience I think we should enable the sound to be played anywhere, so this line of code should be removed.

Not sure how to use a dimension tag, didn't know that was possible.